planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/ConfiguredProfile.java

```diff
@@ -84,6 +90,36 @@ public void processFeature(SourceFeature sourceFeature, FeatureCollector feature
     }
   }
 
+  @Override
+  public List<VectorTile.Feature> postProcessLayerFeatures(String layer, int zoom,
+    List<VectorTile.Feature> items) throws GeometryException {
+    FeatureLayer featureLayer = findFeatureLayer(layer);
+
+    if (featureLayer.postProcess() == null) {
+      return items;
+    }
+
+    if (featureLayer.postProcess().mergeLineStrings() != null) {
+      var merge = featureLayer.postProcess().mergeLineStrings();
+
+      return FeatureMerge.mergeLineStrings(items,
+        merge.minLength(), // after merging, remove lines that are still less than {minLength}px long
+        merge.tolerance(), // simplify output linestrings using a {tolerance}px tolerance
+        merge.buffer() // remove any detail more than {buffer}px outside the tile boundary
+      );
+    }
+
+    if (featureLayer.postProcess().mergePolygons() != null) {
+      var merge = featureLayer.postProcess().mergePolygons();
+
+      return FeatureMerge.mergeOverlappingPolygons(items,
+        merge.minArea() // after merging, remove polygons that are still less than {minArea} in square tile pixels
+      );
+    }
```

Comment:
There is a chance that there are lines and polygons in the same layer and we want to apply merge to both of them. mergeLineStrings will ignore polygons and mergeOverlappingPolygons will ignore linestrings, so let's change this so that if both are set it applies them sequentially to items and returns the result.

Reply:
Change pushed.